src/js/jagex2/config/IdkType.ts

```diff
@@ -44,7 +44,7 @@ export default class IdkType extends ConfigType {
         } else if (code >= 60 && code < 70) {
             this.heads[code - 60] = dat.g2;
         } else {
-            throw new Error(`Unrecognized idk config code: ${code}`);
+            throw new Error(`Unrecognised config code: ${code}`);
```

Comment:
Maybe I should have explicitly mentioned but should double check all of these are like the Java client.

Reply:
Ah... I actually looked at just the text itself, but it seems they're all just being printed on the java client. Fixed, the mesanim one is made up though.